master/buildbot/steps/shell.py

```diff
@@ -248,7 +248,7 @@ def buildCommandKwargs(self, warnings):
 
         # check for the usePTY flag
         if 'usePTY' in kwargs and kwargs['usePTY'] is not None:
-            if self.workerVersionIsOlderThan("svn", "2.7"):
+            if self.workerVersionIsOlderThan("shell", "2.7"):
```

Comment:
actually, reporter says that this change did matter.
because there is no "svn" command anymore in the worker package
I looked for other lines with the same problem, and there is none.